docs/decisions/0002-footer-customizations.rst

```diff
@@ -0,0 +1,138 @@
+2. Footer Customizations
+########################
+.. The title should be a short noun phrase. For example, "1. Django IDA" or "9. LDAP for Multitenant Integration"; filename should be lowercase with punctuation removed and spaces replaced by dash
+
+Status
+******
+
+**Draft (=> Accepted)**
+
+.. **Accepted** *2023-05-11*
+
+.. Standard statuses
+    - **Draft** if the decision is newly proposed and in active discussion
+    - **Provisional** if the decision is still preliminary and in experimental phase
+    - **Accepted** *(date)* once it is agreed upon
+    - **Superseded** *(date)* with a reference to its replacement if a later ADR changes or reverses the decision
+
+    If an ADR has Draft status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Draft (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
+
+Context
+*******
+
+.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+Before the frontend had been started to get broken up into micro frontends
+(MFEs), legacy UI had a mechanism that allowed a great degree of customization
+for the UI through the `legacy UI theming`_. Apart from changing the styles, it
+was possible to selectively overwrite a layout of a component, without having
+to create and maintain forks of the repositories.
+
+With the existing mechanisms for customizing the MFEs it is not possible to
+change the content or layout of selected components without creating a fork of
+a desired MFE and making the change directly in the code, which requires a lot
+of effort to maintain.
+
+It is important to reduce the gap in customisability between the
+legacy UI and MFEs, while keeping the maintenance costs for the customisation
+options low. Otherwise, the vendors and their clients would have an unpleasant
+decision to make - either to forgo any advanced or niche UI customisations, or
+pay a high recurring cost for maintaining them.
```

Comment:
I think we should just focus on the footer here, since we are not coming up with a solution to solve this for all situations. I think you can focus on the fact that there exists a branding API that can produce a customised footer (/api/branding/v1/footer) and that this isn't used by the new footer, and a similar mechanism would solve a number of usecases.